src/poliastro/twobody/orbit.py

```diff
@@ -1085,25 +1032,16 @@ def apply_maneuver(self, maneuver, intermediate=False):
             res = orbit_new
         return res
 
-    def plot(self, label=None, use_3d=False, static=False):
+    def plot(self, label=None, use_3d=False):
```

Comment:
Check that no lines are removed in this function when pushing again. Once you have solved all requested changes, apply to your code `tox -e check`. This will format your code and you will be almost finished :tada: